app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/Save.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Backend\App\Action;
+use Magento\Framework\App\Request\DataPersistorInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Class Save
+ * @package BodenkoVV\AskQuestion\Controller\Adminhtml\Question
+ */
+class Save extends \Magento\Backend\App\Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::save';
+
+    /**
+     * @var PostDataProcessor
+     */
+    protected $dataProcessor;
+
+    /**
+     * @var DataPersistorInterface
+     */
+    protected $dataPersistor;
+
+    /** @var \BodenkoVV\AskQuestion\Model\QuestionFactory  */
+    private $questionFactory;
+
+    /**
+     * @var \Magento\Cms\Api\PageRepositoryInterface
+     */
+    private $pageRepository;
+
+    /**
+     * @param Action\Context $context
+     * @param PostDataProcessor $dataProcessor
+     * @param DataPersistorInterface $dataPersistor
+     * @param \BodenkoVV\AskQuestion\Model\QuestionFactory $questionFactory
+     */
+    public function __construct(
+        Action\Context $context,
+        PostDataProcessor $dataProcessor,
+        DataPersistorInterface $dataPersistor,
+        \BodenkoVV\AskQuestion\Model\QuestionFactory $questionFactory = null
```

Comment:
It's incorrect to specify the default value as null, in the case of null, this class doesn't work. 
And it will cause incorrect substitution while dependency injection container is building this class. Please remove " = null"